src/components/screens/home-screen.tsx

```diff
@@ -17,27 +18,60 @@ const HomeScreen = () => {
   const users = useAtomValue(usersAtom);
   const userProfile = useAtomValue(userProfileAtom);
   const loggedInUser = users.find((user: User) => user.id === userProfile?.id);
-  // TODO: Uncomment this code when the optIn feature is ready
-  // const isOptIn = loggedInUser?.attributes?.severaUserId;
-  // const balanceCard = developerMode && isOptIn ? <BalanceCard /> : null;
-  const balanceCard = developerMode ? <BalanceCard /> : null;
-  const sprintViewCard = developerMode ? <SprintViewCard /> : null;
-  const vacationsCard = developerMode ? <VacationsCard /> : null;
-  const questionairesCard = developerMode ? <QuestionnaireCard /> : null;
-  
+  const hasSeveraUserId = !!loggedInUser?.attributes?.severaUserId;
+
+  const renderCardWithSkeleton = (title: string, content: React.ReactNode) => (
+    <Grid
+      sx={{
+        background: "#f5f5f5",
+        borderRadius: 1,
+        boxShadow: "0px 2px 8px rgba(0,0,0,0.05)",
+        marginBottom: 2,
+        minHeight: 120,
+        display: "flex",
+        flexDirection: "column",
+        justifyContent: "flex-start"
+      }}
+    >
+      <Grid sx={{ padding: 2 }}>
+        <Grid sx={{ fontWeight: "bold", fontSize: 22 }}>
+          {title}
+        </Grid>
+        {!hasSeveraUserId ? (
+          <>
+            <div style={{ color: "#888", fontSize: 15, padding: "12px 0" }}>
+              {strings.notOptedInDescription.description}
+            </div>
+            <Skeleton variant="rectangular" height={20} sx={{ borderRadius: 1, marginTop: 1, width: "100%" }} />
+          </>
+        ) : (
+          content
+        )}
+      </Grid>
+    </Grid>
+  );
+
   return (
     <Grid container spacing={2}>
       <Grid item xs={12} sm={6}>
-        {balanceCard}
+        {developerMode &&
+          renderCardWithSkeleton(
+            strings.balanceCard.balance,
+            <BalanceCard />
+          )}
         <Grid item xs={12} style={{ marginTop: "16px" }}>
-          {sprintViewCard}
+          {developerMode &&
+            renderCardWithSkeleton(
+              strings.sprint.sprintview,
+              <SprintViewCard />
+            )}
         </Grid>
       </Grid>
       <Grid item xs={12} sm={6}>
-        {vacationsCard}
+        {developerMode && <VacationsCard />}
       </Grid>
       <Grid item xs={12} sm={6}>
-        {questionairesCard}
+        {developerMode && <QuestionnaireCard />}
```

Comment:
Same here, we want these to render for all users, so developerMode condition check can be removed.